src/uct/ib/mlx5/ib_mlx5.c

```diff
@@ -768,9 +769,9 @@ uct_ib_mlx5_select_sl(const uct_ib_iface_config_t *ib_config,
     }
 
     *sl_p = sl;
-    ucs_debug("SL=%u (AR support - %s) was selected on %s,"
+    ucs_debug("SL=%u (AR support - %s) was selected on "UCT_IB_IFACE_FMT","
```

Comment:
i think better not use UCT_IB_IFACE_FMT without UCT_IB_IFACE_ARG

Reply:
fixed